### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by pm3310** [zilliztech/GPTCache#617] [Q&A]
Is this project still active? I see that there has been no commit for the last 4 months. Also, many of the example code snippets are broken. For example: https://gptcache.readthedocs.io/en/latest/bootcamp/openai/chat.html

**Comment by SimFG** [comment]
yes, it's active. However, it is not compatible with the 1.x version of openai. I have already submitted a [PR](https://github.com/zilliztech/GPTCache/pull/614) for the new adaptation. But I don’t seem to have received any feedback, so I plan to merge it and release a new version soon.

**Comment by pm3310** [comment]
thanks @SimFG . One more question, I was trying to find more details about the semantic matching when it comes to caching. As it's model powered, there will be false positives and false negatives. Have you done any analysis with regards to this? Or do I have to find the right threshold value for my use case?

**Comment by SimFG** [comment]
@pm3310 I think you need to set this threshold based on your usage scenario

**Comment by pm3310** [comment]
thanks @SimFG it's not very clear in the docs. Also, there's no mechanism provided to fine tune it. I think this is the most important think in order to minimize false positives. Any pointer here?

**Comment by stephenleo** [comment]
I see the PR has been merged, can please update the release also?

**Reply by SimFG** [reply]
release the dev branch?

**Reply by stephenleo** [reply]
sorry, my bad. It's only been merged to dev. looking forward to the main release

**Reply by SimFG** [reply]
can you share what feature you want to use in the release branch?

**Reply by arvashis** [reply]
Just wondering where are we with support for OpenAI 1.x, want to contribute myself but in a time crunch for a project and thought I could use GPTCache with my 4o assistants.

**Reply by SimFG** [reply]
The dev branch includes some support for openai 1.x, but the usage has changed a bit, see: https://github.com/zilliztech/GPTCache/issues/613. Of course, if you have better suggestions, welcome to contribute.
